scripts/simplify_network.py

```diff
@@ -185,6 +185,7 @@ def _adjust_capital_costs_using_connection_costs(n, connection_costs_to_bus):
             n.generators.loc[costs.index, "capital_cost"] += costs
             logger.info("Displacing {} generator(s) and adding connection costs to capital_costs: {} "
                         .format(tech, ", ".join("{:.0f} Eur/MW/a for `{}`".format(d, b) for b, d in costs.iteritems())))
+    n.generators.loc[costs.index, "added_connection_capital_cost"] = costs
```

Comment:
Almost: I think it needs to go inside the for loop to be logged for all technologies. Otherwise, it'll only log the last.
```suggestion
            n.generators.loc[costs.index, "added_connection_capital_cost"] = costs
```

Reply:
Took the liberty to commit this suggestion directly, as I would like to get the other fix in quickly. Hope you don't mind.

Reply:
This results in somewhat unexpected behaviour, as all entries in `n.generators.loc[costs.index, "added_connection_capital_cost"]` in `elec_s.nc` (or `elec_s_{clusters}.nc`) are `NaN`. 

From what I understand, operations are performed on the `elec.nc` network in `simplify_network.py`, which is later aggregated to the `elec_s.nc`. Adding a `print` statement in the code yields the expected result, but I fail to find the data in any of the output files.

Might the information be lost somewhere during aggregation?

Please correct me, if I'm mistaken anywhere.

Reply:
Ah, you might be right.

Reply:
Ok, now done in a separate file as you initially suggested. Sorry for taking a while to respond to this.